src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/pyproject.toml

```diff
@@ -31,33 +31,35 @@ packages = [
     { include = "ansys", from = "src" },
 ]
 
+[[tool.poetry.source]]
+name = "PyPI"
+priority = "primary"
+
 [[tool.poetry.source]]
 name = "solutions-private-pypi"
```

Comment:
The change to the CI ENV var from solutions to PYANSYS also means that this value - the URL value  - and all of the `source = ''` sections below need to change.

Note that this requires an update to the poetry.lock file to reflect the changes.

Reply:
The PyAnsys PyPI has two feeds: pyansys and ansys-solutions. The URLs are therefore different. So, in the `pyproject.toml` we still need to distinguish both. I just realized that we should keep `POETRY_HTTP_BASIC_SOLUTIONS_PRIVATE_PYPI_USERNAME` and `POETRY_HTTP_BASIC_SOLUTIONS_PRIVATE_PYPI_PASSWORD` in the `build-release.yml` and not replace `SOLUTIONS` by `PYANSYS`. The only thing that changes is that both feeds uses the same token. This simplification is enforced in the setup_environment.py script.

Reply:
ruh roh - does that mean this change was a mistake and needs to be reverted? https://github.com/ansys-internal/solution-applications-actions/pull/99

If the keys are the same, can't we simply use the ENV vars to reference the same values?